Tools/Temp/ChangeEditingContextInput.asset

```diff
@@ -0,0 +1,90 @@
+%YAML 1.1
```

Comment:
It looks like you need to serialize this file. I don't see a "Previous" control in the asset, but I see it in the script. I get errors when I  try to use the tool as a result.